src/containerapp/azext_containerapp/_help.py

```diff
@@ -289,6 +289,15 @@
           az containerapp revision label remove -n MyContainerapp -g MyResourceGroup --label myLabel
 """
 
+helps['containerapp revision label swap'] = """
+    type: command
+    short-summary: Swap a revision label between two revisions with associated traffic weights.
+    examples:
+    - name: Swap a revision label between two revisions..
+      text: |
+          az containerapp revision label swap -n MyContainerapp -g MyResourceGroup --labels myLabel1 myLabel2
```

Comment:
Since we are swapping labels, there isn't really a distinction between source and target (just two labels being swapped)

Reply:
ok , my ask is to make this clear by saying label1 , label 2 - instead of worrying about these cannot be space delimited & causes confusion.
 what happens with error handling if user passes 3 space separated names? Is there a reason this is space separated only? what can't  use comma delimited or some other delimiter? 
Can the label names itself have some special characters? what happens to the command if the label name has special characters or one of the label names is incorrect or doesn't exist?

Reply:
If user passes 3 names, it raises an error stating you must pass two values. We can use comma delimited or some other delimiter, I figured for two it wouldn't make much sense. Label names are not validated on our end, only on API end. If the label name is incorrect or doesn't exist you get an error message stating the label could not be found.